File: flying-admin/router/namespace.go
package router

import (
	"flying-admin/api"
	"github.com/gin-gonic/gin"
)

func InitNamespaceRouter(Router *gin.RouterGroup) {
	NamespaceRouter := Router.Group("namespace")
	{
		//NamespaceRouter.POST("release", v1.Release)                             // 发布Namespace
		NamespaceRouter.POST("createNamespace", api.CreateNamespace)             // 新建Namespace
		NamespaceRouter.DELETE("deleteNamespace", api.DeleteNamespace)           // 删除Namespace
		NamespaceRouter.DELETE("deleteNamespaceByIds", api.DeleteNamespaceByIds) // 批量删除Namespace
		NamespaceRouter.PUT("updateNamespace", api.UpdateNamespace)              // 更新Namespace
		NamespaceRouter.GET("findNamespace", api.FindNamespace)                  // 根据ID获取Namespace
		NamespaceRouter.GET("getNamespaceList", api.GetNamespaceList)            // 获取Namespace列表
		NamespaceRouter.GET("getAppNamespace", api.GetAppNamespace)              // 根据环境获取Namespace
		NamespaceRouter.POST("syncNamespace", api.SyncNamespace)                 //同步配置
		NamespaceRouter.POST("findRemoteNamespace", api.FindRemoteNamespace)     //远程查询配置是否存在
	}
}

File: flying-ui/src/api/namespace.js
import request from '@/router/axios';
export function getAppNamespace(query) {
    return request({
        url: '/namespace/getAppNamespace',
        method: 'get',
        params: query
    })
}
export function findNamespace(query) {
    return request({
        url: '/namespace/findNamespace',
        method: 'get',
        params: query
    })
}
export function createNamespace(appId,name,nodes,format,value) {
    return request({
        url: '/namespace/createNamespace',
        method: 'post',
        data: {
            appId: appId,
            name: name,
            nodes: nodes,
            format: format,
            value: value,

        }
    })
}
export function createRelease(obj) {
    return request({
        url: '/release/createRelease',
        method: 'post',
        data: obj
    })
}
export function updateNamespace(obj) {
    return request({
        url: '/namespace/updateNamespace',
        method: 'put',
        data: obj
    })
}
export function deleteNamespace(id,nodeId) {
    return request({
        url: '/namespace/deleteNamespace',
        method: 'delete',
        data:{
            id:id,
            nodeId:nodeId
        }

    })
}
export function findRemoteNamespace(obj) {
    return request({
        url: '/namespace/findRemoteNamespace',
        method: 'post',
        data: obj

    })
}
export function syncNamespace(obj) {
    return request({
        url: '/namespace/syncNamespace',
        method: 'post',
        data: obj

    })
}